packages/opentelemetry-tracing/test/export/BatchSpanProcessor.test.ts

```diff
@@ -60,16 +60,19 @@ describe('BatchSpanProcessor', () => {
     it('should create a BatchSpanProcessor instance', () => {
       const processor = new BatchSpanProcessor(exporter);
       assert.ok(processor instanceof BatchSpanProcessor);
+      processor.shutdown();
     });
 
     it('should create a BatchSpanProcessor instance with config', () => {
       const processor = new BatchSpanProcessor(exporter, defaultBufferConfig);
       assert.ok(processor instanceof BatchSpanProcessor);
+      processor.shutdown();
```

Comment:
Could these be scoped at `describe` level and put in an afterEach?

Reply:
I tried doing that, but `BatchSpanProcessor` instance is initialized in each test with diffrent params. I got `TypeError: Cannot read property 'shutdown' of undefined` errror.